src/engraving/rendering/single/singledraw.cpp

```diff
@@ -99,6 +101,9 @@ void SingleDraw::drawItem(const EngravingItem* item, draw::Painter* painter)
     case ElementType::EXPRESSION:   draw(item_cast<const Expression*>(item), painter);
         break;
 
+    case ElementType::FERMATA:      draw(item_cast<const Fermata*>(item), painter);
+        break;
+
```

Comment:
FiguredBass will need to be handled here too: even though it is not in any of the palettes by default, the user can add it there by Ctrl+Shift+dragging it into a palette.

Reply:
thank you, added